dev_docs/contributing/documentation.mdx

```diff
@@ -0,0 +1,195 @@
+---
+id: kibDocumentation
+slug: /kibana-dev-docs/contributing/documentation
+title: Documentation
+summary: Writing documentation during development
+date: 2022-03-01
+tags: ['kibana', 'onboarding', 'dev']
+---
+
+Docs should be written during development and accompany PRs when relevant. There are multiple types of documentation, and different places to add each.
+
+## End-user documentation
+
+Documentation about user facing features should be written in [asciidoc](http://asciidoc.org/) at [https://github.com/elastic/kibana/tree/main/docs](https://github.com/elastic/kibana//tree/master/docs)
+
+To build the docs, you must clone the [elastic/docs](https://github.com/elastic/docs) repo as a sibling of your Kibana repo. Follow the instructions in that project’s [README](https://github.com/elastic/docs#readme) for getting the docs tooling set up.
+
+To build the docs:
+
+```bash
+node scripts/docs.js --open
+```
+
+## REST APIs
+REST APIs should be documented using the following recommended formats:
+
+- [API doc template](https://raw.githubusercontent.com/elastic/docs/master/shared/api-ref-ex.asciidoc)
+- [API object definition template](https://raw.githubusercontent.com/elastic/docs/master/shared/api-definitions-ex.asciidoc)
+
+## Developer documentation
+
+Developer documentation can be segmented into two types: internal plugin details, and information on extending Kibana. Our [Kibana Developer Guide](https://docs.elastic.dev/kibana-dev-docs/getting-started/welcome) is meant to serve the latter. The guide can only be accessed internally, though the raw content is public in our [public repository]((https://github.com/elastic/kibana/tree/main/dev_docs)). We hope to eventually make this system public for the benefit of external developers as well.
```

Comment:
I attempted to simplify the statement about the rendered docs being available only internally by adding the phrase "at the moment" which alludes to our intention of making this public eventually.

```suggestion
Developer documentation can be segmented into two types: internal plugin details, and information on extending Kibana. Our [Kibana Developer Guide](https://docs.elastic.dev/kibana-dev-docs/getting-started/welcome) is meant to serve the latter. The guide can only be accessed internally at the moment, though the raw content is public in our [public repository]((https://github.com/elastic/kibana/tree/main/dev_docs)).
```